model/clusters_mgmt/v1/aws_sts_policies_type.model

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an sts policies for rosa cluster
+class AWSSTSPolicies {
+
+	// Policy Details
+	PolicyDetails String
```

Comment:
The `Policy` prefix in these names doesn't add anything; it is already clear that this is a policy. I'd suggest to rename these fields to just `Details` and `Type`.